source/grammar/qasm3.g4

```diff
@@ -464,7 +464,7 @@ externDeclaration
     ;
 
 // if have function call w/ out args, is ambiguous; may get matched as identifier
-kernelOrSubroutineCall
+externOrSubroutineCall
     : Identifier ( LPAREN expressionList? RPAREN )?
```

Comment:
Are parentheses required in the case of no arguments? I would argue that they should be...

Reply:
+1 for parentheses to remove the ambiguity with indexIdentifier: Identifier ( LBRACKET expressionList RBRACKET )?

Reply:
I would be okay making parentheses mandatory.